generators/client/templates/angular/webpack/_webpack.common.js

```diff
@@ -124,6 +125,15 @@ module.exports = function (options) {
                 $: "jquery",
                 jQuery: "jquery"
             }),
+            new MergeJsonWebpackPlugin({
+                output:{
+                    groupBy:[
+<%_ for (language in languages) { _%>
+                        { pattern:"./src/main/webapp/i18n/<%= languages[language] %>/*.json", fileName:"./src/main/webapp/i18n/<%= languages[language] %>/all.json" },
```

Comment:
Content is always served from `build/target` folder for angular 2+ as we compile TS to js into the `target|build/www` folder so you shouldnt have any difference at all, but I guess the problem here would be that the json are not bundled into the app at compile time there is always a one time http request getting the `all.json`. So we need to bundle the json with the main app bundles. But if you think it can be done later then i'm ok with your current solution